apidoc/Titanium/UI/WebView.yml

```diff
@@ -437,6 +595,13 @@ properties:
     availability: creation
     default: true
 
+  - name: disableZoom
```

Comment:
Yes, Android's zoom controls (such as pinch-zoom and its hover +/- buttons) can be disabled.

I prefer the `enableZoomControls` property name as well. Most frameworks typically use the term "enable" instead of "disable" when naming properties/methods.